projects/south-america/.env

```diff
@@ -1,2 +0,0 @@
-COMPOSE_PROJECT_NAME=pelias
-DATA_DIR=/tmp/pelias/south-america
```

Comment:
Please restore this file, an `.env` file should be included in each project directory.

Reply:
You can simply add this to the contents of the file:

```bash
COMPOSE_PROJECT_NAME=pelias
DATA_DIR=./data
```

Reply:
It's done! Restored .env file at projects/south-america/.env ✅